mypy.ini

```diff
@@ -24,3 +23,9 @@ ignore_missing_imports = True
 
 [mypy-shapely.*]
 ignore_missing_imports = True
+
+[mypy-pystac.*]
+ignore_missing_imports = True
+
+[mypy-stactools.*]
+ignore_missing_imports = True
```

Comment:
I wasn't getting errors for those. And those packages now publish typing info.
What's going on here?

Reply:
I was getting a bunch of mypy import errors, but rebuilt env and now all is well. Reversed my mypy commits.